src/components/Page.js

```diff
@@ -16,6 +17,20 @@ function Page({ title, body, path }) {
                   </Button>
                 </span>
               )}
+              {create_button && (
+                <span className="edit-button">
+                  <Link
+                    to={{
+                      pathname: '/notifications',
+                      hash: "create-new-notification"
+                    }}
+                  >
+                    <Button>
+                      Create a new notification
```

Comment:
I like the fact that you are extending `Page` to support adding a button next to the title, but I think it would make sense to be able to dynamically define the content of the button: label, pathname and hash, so we can reuse this for all the pages if need be.

Reply:
Thanks. ;D

Good point! I'll reimplement the `create` button like what we did for `edit`.